### PR TITLE
PR template for new PEtab problems

### PR DESCRIPTION
**Pull request by dilpath**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/new_petab_problem.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Checklist
+- [ ] Information about the model is in the SBML metadata
+    - [ ] Reference to the original paper, and any other relevant references
+    - [ ] A brief model description (one or two sentences)
+    - [ ] A brief description of the data (one or two sentences)
+    - [ ] Any differences between the implementation and the original publication
+- [ ] Information is in the model's GitHub issue
+    - [ ] Any differences between the implementation and the original publication
+    - [ ] Experience of fitting / uncertainty analysis (e.g. optimizer used, hyperparameters)
```
Comment (dweindl) [inline .github/PULL_REQUEST_TEMPLATE/new_petab_problem.md:9]: "how `nominalParameters` were obtained"? I  guess it's in there, but might be good to be more explicit?

**Review by dweindl — APPROVED** [review]
:+1:

**Review by elbaraim — COMMENTED** [review]
only a small point I have

File: .github/PULL_REQUEST_TEMPLATE/new_petab_problem.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    - [ ] A brief model description (one or two sentences)
+    - [ ] A brief description of the data (one or two sentences)
+    - [ ] Any differences between the implementation and the original publication
+- [ ] Information is in the model's GitHub issue
+    - [ ] Any differences between the implementation and the original publication
+    - [ ] Experience of fitting / uncertainty analysis (e.g. optimizer used, hyperparameters, reproducibility of best fit)
+    - [ ] Source of nominal parameters (e.g.: taken from the original publication, or from your own fitting)
+- [ ] PEtab files
+    - [ ] A "simulated data" measurement table is included, using the nominal parameters
+    - [ ] A visualization table is included, that can be used with the simulate data to reproduce figures from the original publication
```
[the remark this answers is not in the document]
Comment (dilpath) [reply .github/PULL_REQUEST_TEMPLATE/new_petab_problem.md:13]: I think so, helps a lot with checking that a model can reproduce a published result. If the only argument against is that current models do not have this, an issue could be opened?

[the remark this answers is not in the document]
Comment (elbaraim) [reply .github/PULL_REQUEST_TEMPLATE/new_petab_problem.md:13]: That's the only issue, yes. That not all current models have this. For consistency, the decision could be:
- open the issue to complete the missing visu files
- remove the mandatory field in the readme note that you created

[the remark this answers is not in the document]
Comment (dilpath) [reply .github/PULL_REQUEST_TEMPLATE/new_petab_problem.md:13]: I would opt for opening the issue but keeping this mandatory (still consistent I think, hence the issue). Would this be fine?

**Review by elbaraim — APPROVED** [review]
👍🏼

**Pull request merged**